### PR TITLE
Vs/cc test

### PR DESCRIPTION
**Pull request by vsangereanMOZ**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Bugzilla bug ID

**ID:** 1895894 
**Link:** https://bugzilla.mozilla.org/show_bug.cgi?id=1895884

## Type of change

Please delete options that are not relevant.

- [x] New Test

# How does this resolve / make progress on that bug?

The test is completed.

# Screenshots / Explanations

N/A

# Comments / Concerns

May need to extract the function that adds all card details in new CC form

### CODE REVIEW

File: tests/form_autofill/test_create_new_cc_profile.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    # save CC data by using the fake data
+    credit_card_sample_data = util.fake_credit_card_data()
+    about_prefs_cc_popup = AboutPrefsCcPopup(driver, iframe)
+
+    # add a new CC profile
+    about_prefs_cc_popup.get_element(
+        "cc-popup-button", labels=["autofill-manage-add-button"]
+    ).click()
+    about_prefs.actions.send_keys(credit_card_sample_data.card_number).perform()
```
Comment (ben-c-at-moz) [inline tests/form_autofill/test_create_new_cc_profile.py:39]: Would you check with @sarinali on whether there's already a function that does all these tabs? I want to say there is and that might make the code easier to read...

**Review by ben-c-at-moz — CHANGES_REQUESTED** [review]

File: tests/form_autofill/test_create_new_cc_profile.py (hunk 1 above)
Comment (vsangereanMOZ) [reply tests/form_autofill/test_create_new_cc_profile.py:39]: There isn't one yet. I tried to create it but found it to difficult based on my knowledge so far. 
I would hope you guys will guide me through this.

Comment (sarinali) [reply tests/form_autofill/test_create_new_cc_profile.py]: implemented and fixed!

**Review by ben-c-at-moz — APPROVED** [review]
looks good, let's merge this

**Comment by ben-c-at-moz** [comment]
I'm breaking my "don't merge other people's stuff rule" right now, sorry.

**Pull request merged**